NEWS.md

```diff
@@ -1,4 +1,41 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
```

Comment:
the latest version of data-import-converter-storage interface is 1.4, we should update to this version

Reply:
Version updated